packages/larva-patterns/modules/story-grid/story-grid.twig

```diff
@@ -0,0 +1,9 @@
+<div class="story-grid lrv-u-padding-tb-1 lrv-u-padding-tb-2@desktop // {{ story_grid_classes }}">
+	<ul class="{{ story_grid_grid_classes }} // lrv-a-unstyle-list lrv-a-space-children-horizontal lrv-a-space-children--050 lrv-u-margin-b-050 u-letter-spacing-012">
```

Comment:
Remove the u-letter-spacing utility - that doesn't exist in the larva-css (anything w/o an lrv- namespace is a project-level utility.